composer.json

```diff
@@ -24,6 +24,7 @@
         "symfony/filesystem": "^3.4|^4.3|^5.0",
         "symfony/finder": "^3.4|^4.3|^5.0",
         "symfony/framework-bundle": "^3.4|^4.3|^5.0",
+        "symfony/mime": "^4.3|^5.0",
```

Comment:
did we miss to explicitly require this, and symfony 5.1 drops some indirect requiring of it?

Reply:
as you can see [here](https://github.com/symfony/symfony/commit/11cef32ff7d6cf66f74d5e25a41ab2afa751492f) symfony/mime was set as an optional dependency so it's needed to explicitly require it since SimpleMimeTypeGuesser depends on it.

Reply:
There is a related issue, might I add: https://github.com/liip/LiipImagineBundle/issues/1287

Reply:
that's correct.

Reply:
(i am not active on this project, but have merge rights as i work at liip, so want to help out to merge this)

thanks for the pointers! reading #1287, i am unsure how this PR fixes the issue? don't we also need to change to use the new or old mime type class depending on which is available?

Reply:
This is already done in `SimpleMimeTypeGuesser`. The only issue is that since now `symfony/mime` is set as an optional dependency for `symfony/http-foundation` we need to explicitly require it. So far till symfony 5.1 this wasn't required since it was pulled from the main `symfony/framework-bundle` dependency.

We highly appreciate your help.

Reply:
thanks for the explanation!